src/main/java/org/jenkinsci/plugins/workflow/job/WorkflowRun.java

```diff
@@ -874,10 +874,24 @@ private void finish(@Nonnull Result r, @CheckForNull Throwable t) {
                     }
                     fetchedExecution.onLoad(new Owner(this));
                     if (this.completed != Boolean.TRUE) {
-                        // Defer the normal listener to ensure onLoad can complete before finish() is called since that may
-                        // need the build to be loaded and can result in loading loops otherwise.
-                        fetchedExecution.removeListener(finishListener);
-                        fetchedExecution.addListener(new GraphL());
```

Comment:
Note to self: minor merge conflict with https://github.com/jenkinsci/workflow-job-plugin/pull/27.